src/components/Stats.tsx

```diff
@@ -0,0 +1,199 @@
+import * as React from "react";
+import { Panel, Button, Tabs } from "library-simplified-reusable-components";
+import { LibraryData } from "../interfaces";
+import { ResponsiveContainer, PieChart, Pie, Sector, BarChart, Bar, CartesianGrid, XAxis, YAxis, Tooltip, Legend } from "recharts";
+
+export interface StatsProps {
+  libraries?: LibraryData[];
+}
+
+export interface StatsState {
+  expanded: boolean;
+  copied: boolean;
+  styled: boolean;
+}
+
+export default class Stats extends React.Component<StatsProps, StatsState> {
+  private statsRef = React.createRef<any>();
+
+  constructor(props: StatsProps) {
+    super(props);
+    this.sortLibraries = this.sortLibraries.bind(this);
+    this.renderListView = this.renderListView.bind(this);
+    this.renderChartView = this.renderChartView.bind(this);
+    this.copy = this.copy.bind(this);
+    this.toggleExpanded = this.toggleExpanded.bind(this);
+    this.toggleFormatting = this.toggleFormatting.bind(this);
+    this.state = { expanded: false, copied: false, styled: true };
+  };
+
+  render(): JSX.Element {
+    let sorted = this.sortLibraries();
+    let tabItems = {
+      "List": this.renderListView(sorted),
+      "Charts": this.renderChartView(sorted)
+    };
+
+    return (
+      <Panel
+        id="stats"
+        headerText={"Aggregate Data"}
+        content={
+          <div className="stats-panel">
+            <Tabs items={tabItems} uniqueId="stats-tabs"/>
+          </div>
+        }
+      />
+    );
+  }
+
+  sortLibraries() {
+    let sorted = {"production": [], "testing": [], "cancelled": []};
+    this.props.libraries && this.props.libraries.forEach((l) => {
+      if (Object.values(l.stages).some(v => v === "cancelled")) {
+        sorted.cancelled.push(l);
+      }
+      else if (Object.values(l.stages).some(v => v === "testing")) {
+        sorted.testing.push(l);
+      }
+      else {
+        sorted.production.push(l);
+      }
+    });
+    return sorted;
+  }
+
+  renderListView(sorted) {
+    let buttons = [
+      <Button
+        key="copy"
+        callback={this.copy}
+        content={this.state.copied ? "Copy Again" : "Copy"}
```

Comment:
The "Copy Again" is a good indicator to the user to let them know that their text was copied, but a more common pattern is a small notification saying "Copied to clipboard".